sdk/keyvault/tests-mgmt.yml

```diff
@@ -0,0 +1,15 @@
+trigger: none
+
+stages:
+  - template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
```

Comment:
I'm trying to have use use consistent absolute paths for template includes in our configs, if you don't mind updating to:

```
- template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
```

Reply:
will do!